Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -59,6 +59,7 @@ def runSelected(opt):
                      135.4, #Run 2 Zee ttbar
                      10021.0, #2017 tenmu
                      10024.0, #2017 ttbar
+                     10424.0, #2017 NewPix ttbar
```

Comment:
@slava77 I admit that I forgot about the 25xxx used for 2015/2016. Well, if all pending PRs were merged, 2023 WF numbers would extend up to 20000+200\*4\*4 = 23200 (D1+D2+D3+D4, with 200 slots reserved each, multiplied by 4 (normal, PU, timing, timing+PU)). Hopefully we won't expand too much beyond that before some consolidation occurs, but in the worst case, we can manually skip 25xxx.